styles.css

```diff
@@ -139,3 +139,33 @@ div.dataview-error-box {
 .dataview.small-text::after {
 	content: ")";
 }
+
+/****************/
+/** Pagination **/
+/****************/
+
+.pagination .page-link.disabled {
+    pointer-events: none;
+    color: gray;
```

Comment:
I suspect that the hard-coded colours here and on line 156 will be problematic for some users, depending on the theme and light/dark...

I see that earlier in this file, standard obsidian variable names are used for colours.

Reply:
Thanks, I didn't think it it